components/Dices.js

```diff
@@ -0,0 +1,78 @@
+import useStore from '../hooks/useStore';
+import styled, { css } from 'styled-components';
+import { useState } from 'react';
+
+export default function Dices() {
```

Comment:
Just a really tiny comment: I would prefer calling this component `Dice`, because it's showing one dice not all of them at once. Just my personal preference 😉